src/drivers/mysql/drv_mysql.c

```diff
@@ -997,10 +997,17 @@ db_error_t mysql_drv_query(db_conn_t *sb_conn, const char *query, size_t len,
     {
       DEBUG("mysql_drv_query - Found dropped query from plugin", NULL);
       rs->counter = SB_CNT_GUARANTEED_CAPACITY_DROP;
+      rs->nrows = 0;
```

Comment:
what is `nrows`, `nfields`, and `ptr` for?

Reply:
`rs` is the object we access from lua. At some point, I was getting segfault accessing rs's different values. If we do not set `rs->xyz`, I thought because it is uninitialized, it might be the reason.
But maybe this is not necessary anymore because I only access `rs->counter` which is already set.